packages/styled-components/.prettierignore

```diff
@@ -0,0 +1 @@
+src/utils/errors.js
```

Comment:
I created a `.prettierignore` under `packages/styled-components`. 
This is to prevent the CI from failed.
https://github.com/styled-components/styled-components/actions/runs/5817983702/job/15773544427